src/App.tsx

```diff
@@ -41,10 +41,17 @@ const routes = [{
 // React Router
 const router = createBrowserRouter(routes);
 
-function App() {
-  return (
-    <RouterProvider router={router} />
-  )
+class App extends Component {
```

Comment:
I believe this can be done by modifying `routes` instead.

Reply:
let me look into that...